test/uk/gov/hmrc/eusubsidycompliancefrontend/controllers/UndertakingControllerSpec.scala

```diff
@@ -35,17 +39,40 @@ class UndertakingControllerSpec extends ControllerSpec
   with AuthSupport
   with JourneyStoreSupport
   with AuthAndSessionDataBehaviour
-  with JourneySupport {
+  with JourneySupport
+  with RetrieveEmailSupport
+  with SendEmailSupport  {
   val mockEscService = mock[EscService]
 
   override def overrideBindings = List(
     bind[AuthConnector].toInstance(mockAuthConnector),
     bind[Store].toInstance(mockJourneyStore),
     bind[EscService].toInstance(mockEscService),
-    bind[JourneyTraverseService].toInstance(mockJourneyTraverseService)
+    bind[JourneyTraverseService].toInstance(mockJourneyTraverseService),
+    bind[SendEmailService].toInstance(mockSendEmailService),
+    bind[RetrieveEmailService].toInstance(mockRetrieveEmailService)
   )
 
-  val controller: UndertakingController = instanceOf[UndertakingController]
+  override def additionalConfig = super.additionalConfig.withFallback(
+    Configuration(
+      ConfigFactory.parseString(
+        s"""
+           |
+           |play.i18n.langs = ["en", "cy", "fr"]
```

Comment:
fr?

Reply:
just wanted to add a test if we select any language other than en and cy , it should give error. So added fr code for french